cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -0,0 +1,145 @@
+describe('Set a landing page from the admin portal', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/admin/customization/pages');
+      });
+    });
+  });
+
+  it('should set a landing page when no other landing page exists', () => {
+    cy.findAllByRole('link', { name: 'Edit' }).first().click();
+    cy.findAllByRole('checkbox', { name: "Use as 'Locked Screen'" })
+      .first()
+      .check();
+    cy.findAllByRole('button', { name: 'Update Page' }).first().click();
+
+    // Verify that the form has submitted and the page has changed to the confirmation page
+    cy.url().should('contain', '/admin/customization/pages');
+
+    cy.findAllByRole('img', { name: 'Current locked screen' })
+      .first()
+      .should('be.visible');
+  });
+
+  it('should overwrite the landing page when choosing to set a new landing page', () => {
+    cy.findAllByRole('link', { name: 'Edit' }).first().click();
+
+    // Set landing page
+    cy.findByRole('main').within(() => {
+      cy.findAllByRole('checkbox', { name: "Use as 'Locked Screen'" })
+        .first()
+        .check();
+
+      cy.findAllByRole('button', { name: 'Update Page' }).first().click();
+    });
+
+    // Retrieve the title of the landing page
+    let landingPageTitle;
+    cy.findByRole('main').within(() => {
+      cy.findAllByTestId('page').should((elements) => {
+        for (let i = elements.length - 1; i >= 0; i--) {
+          const el = elements[i];
+          const isLandingPage =
+            el.getElementsByClassName('crayons-icon').length > 0;
+          if (isLandingPage) {
+            landingPageTitle = el.querySelector('a').innerHTML;
+          }
+        }
+      });
+
+      cy.findAllByRole('link', { name: 'Edit' }).first().click();
+    });
+
```

Comment:
```suggestion
cy.findByRole('checkbox', { name: "Use as 'Locked Screen'" })
```

We change page after the Edit click, so we need to wait on something unique to the page - I think the checkbox will do!